ros_ws/src/dataset_generation/CMakeLists.txt

```diff
@@ -169,6 +178,13 @@ target_link_libraries(extract_point_cloud_objects_node
   ${Eigen_LIBRARIES}
 )
 
+target_link_libraries(filter_point_clouds_node
+  ${Boost_LIBRARIES}
+  ${PCL_LIBRARIES}
+  ${Eigen_LIBRARIES}
+  -lpcl_filters
```

Comment:
hmmm you shouldn't need to do this technically. It should be included in `${PCL_LIBRARIES}`. I'll take a look

Reply:
It causes linking error that is why I had to include this explicitly.